rollup.config.js

```diff
@@ -17,47 +15,48 @@ const banner = `/*!
 module.exports = [
 	{
 		input: 'src/index.js',
-		output: {
-			name: 'Chart.Financial',
-			file: `dist/${pkg.name}.js`,
-			banner: banner,
-			format: 'umd',
-			indent: false,
-			globals: {
-				'chart.js': 'Chart'
+		output: ['.js', '.min.js'].map((suffix) => {
+			const config = {
+				name: 'Chart.Financial',
+				file: `dist/${pkg.name}${suffix}`,
+				banner: banner,
+				format: 'umd',
+				indent: false,
+				plugins: [],
+				globals: {
+					'chart.js': 'Chart',
+					'chart.js/helpers': 'Chart.helpers',
+				},
+			};
+
+			if (suffix.match(/\.min\.js$/)) {
+				config.plugins.push(
+					terser({
+						output: {
+							comments: /^!/,
```

Comment:
can you add a comment explaining what this does? I looked briefly at the terser docs and think it keeps comments beginning with `!`? which comments would those be?

Reply:
Those would generally be the license comment blocks, including dependencies. Normally we are required to include those in the built output.

Reply:
I don't think this option makes any change to the output though, so maybe its not needed.

From terser docs:

> By default this works like Google Closure, keeping JSDoc-style comments that contain "@license" or "@preserve".